src/Microsoft.ML.Data/Data/SchemaDefinition.cs

```diff
@@ -17,50 +17,76 @@ namespace Microsoft.ML.Data
     [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
     public sealed class KeyTypeAttribute : Attribute
     {
-        // REVIEW: Property based, but should I just have a constructor?
+
+        /// <summary>
+        /// Marks member as <see cref="KeyType"/>.
+        /// </summary>
+        public KeyTypeAttribute()
+        {
+
```

Comment:
Nit: if you could remove the line in between the braces and add lines between constructors.